content/blog/query-kubernetes/index.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Introducing Pulumi Query
+authors: ["joe-duffy"]
+tags: ["Pulumi-News", "Kubernetes"]
+meta_desc: "Today we announced Pulumi Pulumi Query, an SDK for programmatically querying cloud resources."
+date: "2019-11-14"
+meta_image: "pulumi-crosswalk-k8s.png"
+---
+
+It is too hard to answer simple questions about Kubernetes resources. Questions like:
+
+* How many distinct versions of MySQL are running in my cluster?
+* Which Pods are scheduled on nodes with high memory pressure?
+* Which Pods are publicly exposed to the internet via a load-balanced Service?
+
+Each of these questions would normally be answered by invoking `kubectl` multiple teams to list
+resources of each type, and manually parsing the output to join it together into a single report.
+
+Today, we're introducing Pulumi Query for Kubernetes, which aims to simplify scenarios like this.
```

Comment:
Pretty rough transition - can we make this more natural?

Reply:
Examples like these motivated us to work on Pulumi Query for Kubernetes, which aims to greatly simplify scenarios like the ones mentioned above.

Reply:
Pulumi aims to simplify common Kubernetes operations and scenarios such as these by introducing Pulumi Query for Kubernetes.